docs/config.rst

```diff
@@ -67,4 +67,31 @@ All configuration options above in the general ``[gcp]`` may be used here. Addit
     For the Pub/Sub plugin, ``keyfile`` is not required when running against the `Pub/Sub Emulator <https://cloud.google.com/pubsub/docs/emulator>`_ that Google provides.
 
 
+gcp.gce
+~~~~~~~
+
+All configuration options from the general ``[gcp]`` section may be used here.
+
+Additional plugin-specific configuration is needed:
+
+.. option:: zone="STR"
+
+    `Required`: ID of a Cloud DNS managed zone that instance resource records should belong to.
+
+.. option:: metadata_blackklist=[["STR","STR"],["STR","STR"]]
+
+    `Optional`: List of key-value pairs that will be used to filter out unwanted GCE instances by instance metadata. Note that both the key and the value must match for an instance to be filtered out.
+
+.. option:: tag_blacklist=["STR","STR"]
+
+    `Optional`: List of network tags that will be used to filter out unwanted GCE instances.
```

Comment:
It might be handy to link to [network tags](https://cloud.google.com/vpc/docs/add-remove-network-tags) and [metadata](https://cloud.google.com/compute/docs/storing-retrieving-metadata).

Reply:
Good idea, thanks for providing the links, too!